dkist/io/dask/tests/test_file_manager.py

```diff
@@ -157,3 +157,16 @@ def test_reprs(file_manager):
     assert str(len(sliced_sea)) in repr(sliced_sea)
     assert str(sliced_sea.shape) in repr(sliced_sea)
     assert str(sea) in repr(sliced_sea)
+
+
+@pytest.mark.accept_cli_dataset
+def test_file_manager_slice_slice(large_visp_dataset):
+    stokesI = large_visp_dataset[0]
+    scan = stokesI[..., 0]
```

Comment:
Would it be worth adding a test here which tests the case where the second slice slices the files?